sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -158,11 +157,11 @@ class ODataV4Format(object):
     def __init__(self, json_object):
```

Comment:
should `json_object` be `Dict[str, Any]` since thats how we are treating it ?

Reply:
Fair, doing it